book/listings/actions/5/resources/resources.gresource.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
```

Comment:
Would be nice to have an .editorconfig here to ensure all those and future files are following the same conventions.

Reply:
Can that be enforced with the CI?

Reply:
It can't but VSCode/Builder uses those to enforce things for newly written files/changes to existing ones which is good enough